requirements.txt

```diff
@@ -8,6 +8,7 @@ transformers >= 4.38.0  # Required for Gemma.
 xformers == 0.0.23.post1  # Required for CUDA 12.1.
 fastapi
 uvicorn[standard]
+peft == 0.8.2
```

Comment:
Given that this is only used for testing, it should go to `requirements-dev.txt`